README.md

```diff
@@ -29,6 +29,8 @@ npm install
 npm start
 ```
 
+> To run on a different port, set the `PERFHTML_PORT` env variable to something else, eg `PERFHTML_PORT=1234 npm start`
```

Comment:
so, I just merged a PR that moves our project to Yarn instead of npm. So could you please rebase and then change "npm" here to "yarn" ?